pkg/testutil/testutil.go

```diff
@@ -24,17 +24,18 @@ func CaptureStdout(f func()) string {
 	out := make(chan string)
 	wg := new(sync.WaitGroup)
 	wg.Add(1)
+	var ioCopyErr error
 	go func() {
 		var buf bytes.Buffer
+		_, ioCopyErr = io.Copy(&buf, reader)
 		wg.Done()
```

Comment:
Two things:

1. To make it resilient to the case when `io.Copy` panicked, shall we instead do `defer wg.Done()` BEFORE `io.Copy`?
Doing so alone would result in `out <- buf.String()` hanging forever. To prevent that...
2. We might also change `out := make(chan string)` to `out := make(chan string, 1)`

Not sure if it happens in practice, but we'd better prepare for panicking anyway because debugging a hanging issue is so hard!